tutorials/scripting/godot_cpp/about_godot_cpp.rst

```diff
@@ -0,0 +1,96 @@
+.. _doc_about_godot_cpp:
+
+About godot-cpp
+===============
+
+`godot-cpp <https://github.com/godotengine/godot-cpp>`__ are the official C++ GDExtension bindings, maintained
+as part of the Godot project.
+
+godot-cpp is built with the :ref:`GDExtension system <doc_gdextension>`, which allows access to Godot in almost the
+same way as :ref:`modules <doc_custom_modules_in_cpp>`: A lot of `engine code <https://github.com/godotengine/godot>`__
+can be used in your godot-cpp project almost verbatim.
```

Comment:
```suggestion
can be used in your godot-cpp project almost exactly as it is.
```

Reply:
But verbatim _sounds_ cool! 😄
(will change to what you suggested)